regression/cbmc/Function_Pointer20/test.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+main.c
+
+^\*\*\*\* WARNING: no body for function indirect::fptr_call
```

Comment:
I don't get why there is a warning in this context?  There is only a single thing that `fn_ptr` can point to?